MiniTwit.Entities/Migrations/20200223163409_MoveToDatetime.cs

```diff
@@ -9,19 +9,17 @@ protected override void Up(MigrationBuilder migrationBuilder)
         {
             migrationBuilder.Sql(
                 "ALTER TABLE \"Messages\" ALTER COLUMN \"PubDate\" TYPE timestamp without time zone USING to_timestamp(\"PubDate\")"
-                );
-            
-       
+            );
         }
-       
+
         protected override void Down(MigrationBuilder migrationBuilder)
         {
             migrationBuilder.AlterColumn<int>(
-                name: "PubDate",
-                table: "Messages",
-                type: "integer",
+                "PubDate",
+                "Messages",
+                "integer",
                 nullable: false,
                 oldClrType: typeof(DateTime));
```

Comment:
Why was it necessary to change all the migration files? Please undo this if it was unneeded.

Reply:
My IDE wasn't set up to ignore them in the auto reformatting. It shouldn't make any difference, but I'll try to revert the changes anyway.